README.md

```diff
@@ -150,39 +152,52 @@ if __name__ == "__main__":
 
 ### Creating a Client
 
+**example_client.py**
+
 ```python
 from mcp import ClientSession, StdioServerParameters
 from mcp.client.stdio import stdio_client
 
 # Create server parameters for stdio connection
 server_params = StdioServerParameters(
-    command="path/to/server",
-    args=[], # Optional command line arguments
+    command="python", # Executable
+    args=["example_server.py"], # Optional command line arguments
     env=None # Optional environment variables
 )
 
-async with stdio_client(server_params) as (read, write):
-    async with ClientSession(read, write) as session:
-        # Initialize the connection
-        await session.initialize()
+async def run():
+    async with stdio_client(server_params) as (read, write):
```

Comment:
meanwhile I'm just trying to figure out how to launch a client that doesn't require launching a server in the same script lol

Reply:
It seems like anyio supports tcp (connect_tcp). One option is to follow that route, having the server listen in a port and the client connect to it. That's probably the idea moving forward when running client/servers from different hosts (better if upgraded to TLS).

Reply:
@logan-markewich feel free to move that to a discussion https://github.com/orgs/modelcontextprotocol/discussions